PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+Hello,
+
+Thank you for submitting a pull request, that's awesome! Please add some info (if applicable) to give us some context on the PR.
+
+Things to consider before you submit the PR:
+
+* Has `CHANGELOG.md` been updated? (optional)
+* Are tests passing locally?
```

Comment:
* [ ] `CHANGELOG.md` has been updated  (optional)
* [x] The tests are passing locally

Reply:
I like the checkbox style suggested by @MatkovIvan

Reply:
If we're going to use that style it should be included in the body (not commented out)